content/en/docs/concepts/services-networking/network-policies.md

```diff
@@ -189,6 +189,15 @@ spec:
 
 This ensures that even pods that aren't selected by any other NetworkPolicy will not be allowed ingress or egress traffic.
 
+## SCTP support
+
+{{< feature-state feature-state state="alpha" >}}
```

Comment:
Ah. This is a typo from the shortcode docs. Should just be: `{{< feature-state state="alpha" >}}`. Looks like this is causing netlify to fail.

Reply:
Also, we may want to pin this to 1.12 with `for_k8s_version="v1.12"`